website/static/js/osfHelpers.js

```diff
@@ -456,14 +457,19 @@ var applyBindings = function(viewModel, selector) {
         throw "Can't bind ViewModel to multiple elements."; // jshint ignore: line
     }
     // Ensure that the bound element is shown
-    if ($elem.hasClass('scripted')){
+    if ($elem.hasClass('scripted')) {
         $elem.show();
     }
     // Also show any child elements that have the scripted class
-    $(cssSelector + ' .scripted').each(function(elm) {
+    $(cssSelector + ' .scripted').each(function (elm) {
         $(this).show();
     });
-    ko.applyBindings(viewModel, $elem[0]);
+
+    //Check if bindings have been applied to the element already
+    var isBound = !!ko.dataFor(elem);
+    if(!isBound) {
```

Comment:
I agree, but this is currently the only fix I could come up with without having to rewrite the logic of how profile page is rendered. What happens is that public projects calls render_nodes.mako, which when finished calls render_nodes.js, and it applies its bindings. However, when public components calls render_nodes.mako, it will find the public project nodes, try to reapply the bindings, and fail. Maybe I could attempt another fix entirely contained in render_nodes.js, but as of now because render_nodes.mako is called twice, anything within render_nodes.js will be called twice, specifically applyBindings.